### PR TITLE
Add stale GHA

### PR DESCRIPTION
**Pull request by henningkayser**
Enables a [stale action](https://github.com/marketplace/actions/close-stale-issues) that labels issues and PRs as stale after 45 days, and closes issues after another 45 days. The action also takes care about warning the author appropriately, and it ignores a set of persistent labels. The intention of this is to really only keep issues open that are persistent explicitly (there is a new label for that), or that belong to a roadmap, milestone, etc.. All other issues should require at least some activity within the 3 months of time it would take for this GHA to close them.

Fixes https://github.com/ros-planning/moveit2/pull/2022

### CODE REVIEW

**Review by tylerjw — APPROVED** [review]

**Comment by v4hn** [comment]
I still think this is superficial make-up and the only thing it does is add bloat where maintainer actions are required.
An issue list (even when the issues age) is a valuable resource of user input, not a junkyard that needs automatic discarding. In an ideal world all items in it warrant a manually added "can't reproduce/not enough information/won't fix" label before they are closed without a merged patch. 

I guess you weakened the conditions enough to avoid further complains.

**Comment by AndyZe** [comment]
I thought discussions were supposed to go in the Discussions tab that was the cool new thing a couple months ago  ;)

This type of shuffling of deck chairs amuses me. I hope no valuable old issues get lost because of it.

**Comment by henningkayser** [comment]
Same response as here https://github.com/ros-planning/moveit2/issues/1490#issuecomment-1473867153. 


> I still think this is superficial make-up
yes
> and the only thing it does is add bloat where maintainer actions are required
it automates superficial maintainer actions
> An issue list (even when the issues age) is a valuable resource of user input, not a junkyard
it's both
> that needs automatic discarding
it doesn't, but it helps
> In an ideal world
... 
> all items in it warrant a manually added "can't reproduce/not enough information/won't fix" label before they are closed without a merged patch.
We do have that and use it frequently
> I guess you weakened the conditions enough to avoid further complains.
I hope so. My concern is really not about rejecting valuable issues. It's about requiring people to think about the content and value of their issue first before posting it in the first place. There is a huge amount of content that is along the lines of "we could...", "it would be nice..." that no one ever responds to because it's apparently not important at all. If only all maintainers would post issues of whatever wishes or concerns they have about the code base, the list would be endless and it would still be just as useless. Let's focus on actual problems and action items that are important enough to discuss them.

> I thought discussions were supposed to go in the Discussions tab that was the cool new thing a couple months ago ;)
GitHub discussions are obviously something different than issues but it's hard to argue that issues are not meant for discussions.

**Pull request merged**
